web/themes/custom/asulib_barrio/scss/components/cards.scss

```diff
@@ -25,3 +25,49 @@ extends/_cards.scss
     }
   }
 }
+
```

Comment:
we aren't getting all of this card styling from the uds bootstrap ?

Reply:
I am not sure what "card styling" you are referring to.